src/vscode-bicep/src/utils/logger.ts

```diff
@@ -97,3 +97,7 @@ export function getLogger(): Logger {
 
   return logger;
 }
+
+if (process.env.NODE_ENV === "test") {
+  exports.resetLogger = () => (logger = undefined);
+}
```

Comment:
Any way to avoid this? I'm guessing this is because `logger` is a singleton and is being shared between tests?

Reply:
Yes, this is due to the fact that `logger` is singleton and internal, so it cannot be mocked. This is one of the issues that blocked me, but I cannot think of a better way to make it work for unit tests...